core/src/main/java/io/grpc/Attributes.java

```diff
@@ -56,14 +56,24 @@ private Attributes(Map<Key<?>, Object> data) {
    * Returns set of keys stored in container.
    *
    * @return Set of Key objects.
+   * @deprecated This method is being considered for deprecation, if you feel this method is needed
```

Comment:
s/considered for deprecation/considered for removal/ ? Considering this PR _is_ deprecating it.